tests/foreman/api/test_docker.py

```diff
@@ -1132,7 +1132,7 @@ def setUpClass(cls):
         cls.cr_external = entities.DockerComputeResource(
             name=gen_string('alpha'),
             organization=[cls.org],
-            url=settings.docker.external_url,
+            url='http://{0}:2375'.format(settings.server.hostname),
```

Comment:
It is just a suggestion
I propose to comment out that property at all in the file:
```
# This will be used by tests that test external docker daemon.
external_url=http://localhost:2375
```
Add corresponding logic into settings module:
```
if docker_url is None:
    docker_url = 'http://{0}:2375'.format(settings.server.hostname)
```
So if user want to use its own location, so he can do that easily, if not our new logic will be used instead